src/test/feature/Ranger/test_ranger.cpp

```diff
@@ -24,19 +24,34 @@
 
 #include "lib/command.h"
 #include "lib/gpfdist.h"
-#include "lib/sql_util.h"
 #include "lib/string_util.h"
 
 using std::vector;
 using std::string;
 using hawq::test::SQLUtility;
 using hawq::test::Command;
 
+TestHawqRanger::TestHawqRanger()
+{
+	initfile = hawq::test::stringFormat("Ranger/sql/init_file");
+	rangerHost = getRangerHost();
+}
+
+std::string& TestHawqRanger::getRangerHost()
```

Comment:
I checked rangeruser.py and found that the "-h" option stands for "ranger server host". Since rps won't be installed on ranger server host that usually, so I don't think this is a right way to get ranger server host.